mcdu.html

```diff
@@ -77,35 +78,111 @@
 		}, true);
 	</script>
 	<style>
-		body { font-size: 5rem; margin: 0; padding: 0; background-color: black; color: white; }
-		table { margin: 0; padding: 0 }
-		tr { margin: 0; padding: 0 }
-		td { margin: 0; padding: 0; text-align: center; outline: 1px solid gray; }
-		td:active { background-color: white; color: black }
-		.enter { font-size: 60% }
-		.enter tr:first-child { font-size: 70% }
-		.enter tr:last-child { font-size: 50% }
-		.enter td:first-child, .enter td:last-child { padding: 0 0.55em; }
-		.menu, .arrows .airport { font-size: 40%; line-height: 1em; }
-		.menu td { height: 2.5em; }
-		.arrows td { width: 50%; padding: 0; font-size: 60%; }
-		img { display: block; }
+		body {
+			font-size: 5rem;
+			margin: 0;
+			padding: 0;
+			background-color: black;
+			color: white;
+		}
+
+		table {
+			margin: 0;
+			padding: 0
+		}
+
+		tr {
+			margin: 0;
+			padding: 0
+		}
+
+		td {
+			margin: 0;
+			padding: 0;
+			text-align: center;
+			outline: 1px solid gray;
+		}
+
+		td:active {
+			background-color: white;
+			color: black
+		}
+
+		.enter {
+			font-size: 60%
+		}
+
+		.enter tr:first-child {
+			font-size: 70%
+		}
+
+		.enter tr:last-child {
+			font-size: 50%
+		}
+
+		.enter td:first-child,
+		.enter td:last-child {
+			padding: 0 0.55em;
+		}
+
+		.menu,
+		.arrows .airport {
+			font-size: 40%;
+			line-height: 1em;
+		}
+
+		.menu td {
+			height: 2.5em;
+		}
+
+		.arrows td {
+			width: 50%;
+			padding: 0;
+			font-size: 60%;
+		}
+
+		img {
+			display: block;
+		}
 	</style>
 </head>
+
 <body style="width: 100%">
 	<table class="enter" style="width: 92%">
 		<tr>
-			<td>&nbsp;</td>
```

Comment:
This changed section breaks alignment with the canvas display

Reply:
Fix committed! ff7c4e93f5ec6e9e13a5d4eb2c2ead4a39be0443

Reply:
Looks good, tho the buttons still don't work... not sure what's wrong with that.